src/test/components/TrackMemory.test.js

```diff
@@ -57,13 +57,15 @@ function getSamplesPixelPosition(
  * This test verifies that the memory track can draw a graph of the memory.
  */
 describe('TrackMemory', function () {
-  function setup() {
+  function setup(counterConfig = {}) {
```

Comment:
It could be good to specify the type for the reader, probably in this case: `: $Shape<{| countNumber: number |}>`